include/daisykitsdk/README.md

```diff
@@ -0,0 +1,35 @@
+# Coding structure
+
+
++ **daisykitsdk**
+  + **common** (daisykit::common) common helpers
+    + **io** (daisykit::io): read/write files, config, network
+    + **logging** (daisykit::logging): logging / profilers
+    + **types** (daisykit::types): data structures and types
+    + **visualizers** (daisykit::visualizers): visualizers for object detection, segmentation, keypoints...
+    + **profilers** (daisykit::profilers): profilers to tracking system performance 
+      + node_profiler.h
+      + fps_calculator.h
+      + latency_tracking.h
+      ...
+    + **utils** (daisykit::utils): common utilities
+      + timer.h
+      + filesystem.h
+      ...
+  + **processors** (daisykit::processors) signal/image processors, basic operations to build computational system.
+    + **image_processors**
+    + **signal_processors**
+    + **trackers**
```

Comment:
The difference between ` trackers ` and ` image/signal processors ` is not clear. For example, with ` trackers ` , we track objects on images, right? So `image_processors` deal with images, and `trackers` do the same. Therefore, considering ` trackers ` and ` image_processors ` as individual elements of `processors` seems ambiguous.